### DISCUSSION TITLE
Examples using OpenIddict/Orchard Core

### DISCUSSION THREAD

**Discussion by marcusjm** [jasontaylordev/CleanArchitecture#546] [Q&A]
Due to the new situation with IdentityServer I have moved to using OpenIddict but then I realised OrchardCore had some great features for OpenID/Api Security. Has anyone here created an example where CA would work with either of those? Would be grateful.

Thanks in advance
marcus

**Comment by litan1106** [comment]
I also like to move to OpenIddict because IdentityServer no longer open source and the enterprise - 4 limited apps.

**Reply by marcusjm** [reply]
Orchard Core would be a viable option here.
